app/assets/stylesheets/my_modules/repositories.scss

```diff
@@ -231,30 +231,30 @@
         padding: 0;
       }
 
-      .dataTables_wrapper {
-        height: 100%;
-        position: absolute;
-        width: 100%;
-
-        .dataTables_scroll {
-          height: calc(100% - 146px);
-          padding: 0 24px;
-
-          .dataTables_scrollBody {
-            height: calc(100% - 44px);
-          }
-        }
-
-        .main-actions {
-          padding: 16px 24px 10px;
-        }
-
-        .pagination-row {
-          border-top: $border-default;
-          height: 84px;
-          padding: 0 24px;
-        }
-      }
+      // .dataTables_wrapper {
```

Comment:
Will we leave this here?

Reply:
yes, I assume it will be re-enabled once layout changed